lib/game.rb

```diff
@@ -0,0 +1,96 @@
+require_relative 'board'
```

Comment:
When a tie game occurs, the game does not show that the game is a tie, it just ends its execution.
Optional:
You can add the necessary logic of asking players to start another game.
![Screen Shot 2020-08-08 at 1 46 42 PM](https://user-images.githubusercontent.com/52430951/89717607-c8218200-d97d-11ea-9056-6af086522b1a.png)